presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -327,15 +334,42 @@ public StripeFooter readStripeFooter(StripeInformation stripe, AbstractAggregate
         }
     }
 
-    private Map<Integer, List<RowGroupIndex>> readColumnIndexes(Map<StreamId, Stream> streams, Map<StreamId, OrcInputStream> streamsData)
+    private Map<Integer, List<RowGroupBloomfilter>> readBloomfilterIndexes(Map<StreamId, Stream> streams, Map<StreamId, OrcInputStream> streamsData)
+            throws IOException
+    {
+        ImmutableMap.Builder<Integer, List<RowGroupBloomfilter>> bfIndexes = ImmutableMap.builder();
```

Comment:
Refactored. Took `bloomfilters` as name because we also have the row indexes to prevent confusion.